courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb

```diff
@@ -4,22 +4,25 @@
    "cell_type": "markdown",
```

Comment:
<p>typo. "Constant constant can not be changed, while variables values can be." to "Constant values can not be changed, while variable values can be."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>